### PR TITLE
Technigo-chatbot-Harry and Maria

### PR DESCRIPTION
**Pull request by Groovedharry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,22 @@
-# Project Name
+# Chatbot
 
 Replace this readme with your own information about your project.
 
 Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
```
Comment (annatbg) [inline README.md:6]: It would have been helpful with just a short description of the assignment here.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
 
 ## The problem
-
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
-## View it live
+We had problems with understanding how javascript worked at first. We had a tough time even starting out the code, and it took us along time to actually understand what we did and how to write in javascript.
+
+We approached our problems with an open mind and we asked for help from our group. They came in clutch, they are great! We wanted to fix our problems on our own, as we both felt that we could learn a lot by just trying out stuff. But when we didn't get enywhere, we asked for help and it was very useful.
```
Comment (brunsant) [inline README.md:12]: The lion team is a resource and we should all feel comfortable in asking for support and help. We all feel the same way!

File: README.md [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
-## View it live
+We had problems with understanding how javascript worked at first. We had a tough time even starting out the code, and it took us along time to actually understand what we did and how to write in javascript.
+
+We approached our problems with an open mind and we asked for help from our group. They came in clutch, they are great! We wanted to fix our problems on our own, as we both felt that we could learn a lot by just trying out stuff. But when we didn't get enywhere, we asked for help and it was very useful. 
+
+We did not have much of any plan going into the project as we just follwed to different sections that were setup for us from the technigo brief.
 
+If we had more time we would added more questions to the bot.
+
+We also had a rough start as me (Harry) was ill and did not really have the strength to sit and code for long. I really feel that Maria was very understanding of this and want to thank Maria for this as I felt I dragged her down alittle bit for not being able to put 100% into the project from the start.
```
Comment (annatbg) [inline README.md:18]: We all have bad days/weeks when we struggle, the important thing is that you didn't give up. Remember that you did your best under the circumstances.

File: code/index.html [hunk 4]
```diff
@@ -8,18 +8,18 @@
...
+    <h1>Welcome to Movie-Bob.</h1>
+    <h2>I help you select a movie for tonight.</h2>
     <main>
       <section class="chat" id="chat"></section>
 
       <div class="input-wrapper" id="input-wrapper">
-        <form id="name-form">
-          <input id="name-input" type="text" />
+        <form id="nameForm">
+          <input id="nameInput" type="text" />
```
Comment (brunsant) [inline code/index.html:22]: When you have the id based on the HTML, the convention is to use them as kebab-case. If the id was made in JavaScript then we should use only camelCase. (we did the same and then Matilda pointed that out for us).

File: code/script.js [hunk 5]
```diff
@@ -1,13 +1,19 @@
...
 // Global variables, if you need any, declared here
 
+let currentQuestion = "question1";
 // Functions declared here
 
+
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   if (sender === 'user') {
+    console.log('user is sending')
```
Comment (annatbg) [inline code/script.js:15]: Friendly reminder to remove console.logs before handing in the project. :)

File: code/script.js [hunk 6]
```diff
@@ -17,6 +23,8 @@ const showMessage = (message, sender) => {
       </section>
     `
   } else if (sender === 'bot') {
+    console.log('bot is sending')
```
Comment (annatbg) [inline code/script.js:26]: Another console.log that can be cleaned up.

File: code/script.js [hunk 7]
```diff
@@ -26,22 +34,68 @@ const showMessage = (message, sender) => {
...
+}
+
+const ageQuestion = () => {
+  showMessage('Nice to meet you! How old are you?', 'bot')
+  currentQuestion =`question2`
+}
+
+const resetForm = () => {
+  location.reload();
+};
```
Comment (brunsant) [inline code/script.js:53]: I might be easier to understand if the variables that are not global, follow the conversation flow. Just so we don't break the flow with information that are not necessary at the time.

File: code/script.js [hunk 8]
```diff
@@ -26,22 +34,68 @@ const showMessage = (message, sender) => {
...
+
+const handleNameInput = () => {  //handle the nameinput
+  let message = document.getElementById("nameInput").value;
+
+  if (currentQuestion === "question1") {
+    // const name = message;     //get the value from the user
+    showMessage(message, 'user')
+    inputValue.value = '' //får fältet att bli tomt efter att vi skrivit vårt namn
```
Comment (brunsant) [inline code/script.js:61]: It would be easier to review (for me, Bruna) if all comments are in English.

File: code/script.js [hunk 9]
```diff
@@ -26,22 +34,68 @@ const showMessage = (message, sender) => {
...
+    } else{
+      showMessage('Great! I recommend Once upon a time in America. If you are in the mood for Leo DiCaprio, but not in the mood for blood - you could go with Titianic instead.', `bot`)
+    }
+
+
+
+
+    nameForm.innerHTML = `<button id="reset">Reset</button>`
+    const reset = document.getElementById(`reset`);
+    reset.addEventListener(`click`, resetForm);
```
Comment (annatbg) [inline code/script.js:82]: Great job making the reset-button! Very clean!

**Review by annatbg — COMMENTED** [review]
The bot works well in the browser. The color palette was a good choice for the theme of your bot. You definitely fullfilled the requirements for the project.

Your code is very clean and easy to understand, however, we have pointed to some minor details (console.logs) that could be cleaned up. Even though your code is clean, it would be helpful to have general comments explaining the flow.

To sumarise: we are very impressed by your work this week, keep it up!

/Anna & Bruna

**Pull request closed without merging**